isr_exploration/CTE_Thurs.ipynb

```diff
@@ -0,0 +1,332 @@
+    "E2V_cte_data = readEtravelerCTE(filename)\n"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 7,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "#ITL\n",
+    "# These values are for the ITL detector\n",
+    "\n",
+    "ITL_BOOTCAMP_REPO_DIR= '/project/bootcamp/repo_RTM-011/'\n",
+    "rawname = '5943D'\n",
+    "lsst_num = 'ITL-3800C-083-Dev'\n",
+    "ITL_OFFSET = 20\n",
+    "ITL_Xsize = 509\n",
+    "flipXY = {'C00':(False, True),\n",
+    "          'C01':(False, True),\n",
+    "          'C02':(False, True),\n",
+    "          'C03':(False, True),\n",
+    "          'C04':(False, True),\n",
+    "          'C05':(False, True),\n",
+    "          'C06':(False, True),\n",
+    "          'C07':(False, True),\n",
+    "          'C10':(True, False),\n",
+    "          'C11':(True, False),\n",
+    "          'C12':(True, False),\n",
+    "          'C13':(True, False),\n",
+    "          'C14':(True, False),\n",
+    "          'C15':(True, False),\n",
+    "          'C16':(True, False),\n",
+    "          'C17':(True, False)\n",
+    "         }\n",
+    "\n",
+    "(ITL_low_file, ITL_high_file) = findFitsFiles(ITL_BOOTCAMP_REPO_DIR, rawname, lsst_num)\n",
+    "filename = '/home/cslage/bootcamp-work/isr_exploration/ITL-3800C-083_CTE.txt'\n",
+    "ITL_cte_data = readEtravelerCTE(filename)\n"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Getting Horizontal CTE"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 8,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "def getOverscans(OFFSET, detector):\n",
+    "    overscans2 = {}\n",
+    "    for key, value in flipXY.items():\n",
+    "        bbox = detector[key].getRawHorizontalOverscanBBox()\n",
+    "        if value[0]:\n",
+    "            point = Point2I(bbox.getMinX(), bbox.getMinY())\n",
+    "            ext = Extent2I(bbox.getWidth() + OFFSET, bbox.getHeight())\n",
+    "            new_bbox = Box2I(point, ext)\n",
+    "            overscans2[key] = raw[new_bbox].getMaskedImage().getImage().clone() # since these are views into the pixels, we want a copy so we don't mess with the pixels in the original image\n",
+    "        else:\n",
+    "            point = Point2I(bbox.getMinX() - OFFSET, bbox.getMinY())\n",
+    "            ext = Extent2I(bbox.getWidth() + OFFSET, bbox.getHeight())\n",
+    "            new_bbox = Box2I(point, ext)\n",
+    "        overscans2[key] = raw[new_bbox].getMaskedImage().getImage().clone() # since these are views into the pixels, we want a copy so we don't mess with the pixels in the original image\n",
+    "    return overscans2\n"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 9,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "def compute_Serial_CTE(overscans2, xsize):\n",
+    "    cte_mean = []\n",
+    "    cte_std = []\n",
+    "    for i in range(1,17):\n",
```

Comment:
I think a lot of these hard coded numbers could be explained, or perhaps replaced with getBBox-type references which the previous definition used nicely